scripts/exobsmon_plot.sh

```diff
@@ -24,31 +20,89 @@ if [[ ! -e ${chan_yaml} ]]; then
    exit 2
 fi
 
-#----------------------------------------
-# split $plot_yaml into sat/instr[/plot]
+#-----------------------------------------------------------
+# split $plot_yaml into sat/instr[/plot], minimization, obs
 #
 ${APRUN_PY} ${USHobsmon}/splitPlotYaml.py -i ${plot_yaml} -c ${chan_yaml}
 
-#exit 0
 
-cmdfile="OM_test_jobscript"
->$cmdfile
+#--------------------------------------------------------------
+# Submit OM_sat_plots job if split yields any sat_*.yaml files
+#
+if compgen -G "./sat_*.yaml" > /dev/null; then
```

Comment:
should this be wrapped in a `[[ ]]`?

Reply:
No, I think it's ok as is.  I lifted that from one of the changes the global_workflow team made to one of my RadMon scripts and they don't use `[[ ]]`.  When I added `[[ ]]` I ended up with an error:

`conditional binary operator expected
/scratch1/NCEPDEV/da/Edward.Safford/noscrub/git/obs-monitor/scripts/exobsmon_plot.sh: line 32: syntax error near -G'`

As I understand it compgen is a built-in command, rather than a conditional expression, and since I've redirected all output to `/dev/null` it evaluates as 0 or 1.  If I add `[[ ]]` then a conditional expression is expected but isn't found.

Reply:
Got it, thanks for the explanation

Reply:
I can use a var instead of "./sat*.yaml", I think "${DATA}/sat*.yaml" would work.  I'll test that out.